census/vanilla-tf/README.md

```diff
@@ -0,0 +1,123 @@
+# Census: TensorFlow Vanilla Sample
+
+This sample uses the [TensorFlow](https://tensorflow.org) low level APIs and
+[Google Cloud Machine Learning Engine](https://cloud.google.com/ml) to demonstrate
+the single node and distributed TF vanilla version for Census Income Dataset.
+Setup instructions for the [Cloud ML Engine](https://cloud.google.com/ml/docs/how-tos/getting-set-up).
+
+## Download the data
+Follow the [Census Income
+Dataset](https://www.tensorflow.org/tutorials/wide/#reading_the_census_data) link to download the data. You can also download directly from [here](https://archive.ics.uci.edu/ml/datasets/Census+Income).
+
+ * Training file is `adult.data`
+ * Evaluation file is `adult.test`
+
+
+## Virtual environment
+There are two options for the virtual environments:
+ * Install [Virtual](https://virtualenv.pypa.io/en/stable/) env
+   * Create virtual environment `virtualenv single-tf`
+   * Activate env `source single-tf/bin/activate`
+ * Install [Miniconda](https://conda.io/miniconda.html)
+   * Create conda environment `conda create --name single-tf python=2.7`
+   * Activate env `source activate single-tf`
+
+
+## Install dependencies
+Install the following dependencies:
+ * Install [Cloud SDK](https://cloud.google.com/sdk/)
+ * Install [TensorFlow](https://www.tensorflow.org/install/)
+ * Install [Pandas](http://pandas.pydata.org/pandas-docs/stable/install.html#installing-from-pypi)
+ * Install [gcloud](https://cloud.google.com/sdk/gcloud/)
+
+
+# Single Node Training
+Single node training runs TensorFlow code on a single instance. You can run the exact
+same code locally and on Cloud ML Engine.
+
+## How to run the code
+You can run the code either as a stand-alone python program or using `gcloud`.
+See options below:
+
+### Using local python
+Run the code on your local machine:
+
+```
+python trainer/task.py --train_data_path $TRAIN_DATA_PATH \
+                       --eval_data_path $EVAL_DATA_PATH \
+                       [--max_steps $MAX_STEPS]
+```
+
+### Using gcloud local
+Run the code on your local machine using `gcloud`:
+
+```
+gcloud beta ml local train --package-path trainer \
+                           --module-name trainer.task \
+                           -- \
+                           --train_data_path $TRAIN_DATA_PATH \
+                           --eval_data_path $EVAL_DATA_PATH \
+                           [--max_steps $MAX_STEPS]
+```
+
+### Using gcloud on cloud
```

Comment:
"Using gcloud on cloud" ...I think you want something like "Using Google's Cloud ML Service"

This sounds like the gcloud command is running on the cloud, not that the ML code is running on the cloud.